### PR TITLE
feat: update bakerfi data

### PR DESCRIPTION
**Pull request by ickas**
Update to the new logo and update data because BakerFi is moving to Base:
- Remove the Arbitrum chain
- Add the Base chain
- Add the number of audits

### CODE REVIEW

**Comment by hvasconcelos** [comment]
It is better to keep booth

File: defi/src/protocols/data3.ts [hunk 1]
```diff
@@ -33717,17 +33717,17 @@ const data3: Protocol[] = [
     address: null,
     symbol: "-",
     assetToken: "brETH",
-    url: "https://bakerfi.xyz/",
+    url: "https://bakerfi.xyz",
     description:
       "BakerFi is an innovative DeFi protocol designed to optimize asset growth through efficient staking, re-staking, and recursive strategies, all aimed at enhancing the APY for users. Our platform automates these strategies, streamlining capital allocation, and offers a user-friendly interface accessible to all levels of expertise. Whether you're a novice or seasoned investor, BakerFi provides a seamless experience to maximize your returns.",
-    chain: "Arbitrum",
+    chain: "Base",
```
Comment (Define101) [inline defi/src/protocols/data3.ts:33723]: The current adapter is only tracking Arbitrum chain

Comment (ickas) [reply defi/src/protocols/data3.ts:33723]: > The current adapter is only tracking Arbitrum chain

https://github.com/DefiLlama/DefiLlama-Adapters/commit/6fe67a6d3964d233e981dad393b907d0edcf8a0a

**Comment by Define101** [comment]
We made the changes here https://github.com/DefiLlama/defillama-server/pull/7462
Let us know once base chain as been added

**Comment by ickas** [comment]
> We made the changes here #7462 Let us know once base chain as been added

@Define101 Base chain was added on this commit https://github.com/DefiLlama/DefiLlama-Adapters/commit/6fe67a6d3964d233e981dad393b907d0edcf8a0a, right?

**Comment by Define101** [comment]
Should be live within 2 hours thanks 
https://github.com/DefiLlama/defillama-server/pull/7469

**Pull request closed without merging**